sites/README.md

```diff
@@ -0,0 +1,249 @@
+# Valid properties
+
+List of possible values that can be used
+
+## name
+**Type:** String
+
+This property will be used for all the Kubernetes metadata names. If not specified by default it will be the domain name, replacing the dots with dashes.
+
+Example:
+```
+name: threesixty-canonical-com
+staging:
+  name: threesixty-staging-canonical-com
+```
+
+## domain
+**(Mandatory)**
+
+**Type:** String
+
+This property define the project domain, it will be used for the Service, Deployment and Ingress objects.
+
+Example:
+```
+domain: canonical.com
+```
+
+## image
+**(Mandatory)**
+
+**Type:** String
+
+Set the Docker image you want to use.
+
+Example:
+```
+image: prod-comms.docker-registry.canonical.com/canonical.com
+```
+
+## replicas
+
+**Type:** Int
+
+**Default:** `5`
+
+Number of replicas to use.
+
+Example:
+```
+replicas: 2
+```
+
+## containerPort
+
+**Type:** Int
+
+**Default:** `80`
+
+Port to expose from the container.
+
+Example:
+```
+containerPort: 8080
+```
+
+## useProxy
+
+**Type:** Boolean
+
+**Default:** `True`
+
+This will use our ConfigMap object "proxy-config", you can take a look to this object here: `configmaps/proxy-config.yaml`
```

Comment:
This is no longer accurate is it?

Reply:
No, you are right. I updated the description.